### DISCUSSION TITLE
For the love of God please move Settings dialog where it belongs

### DISCUSSION THREAD

**Discussion by andreszs** [rgthree/rgthree-comfy#401] [Ideas]
The **Settings** shortcut that has been happily placed in the main toolbar as follows

![image](https://github.com/user-attachments/assets/c86dbaed-5925-4a44-a265-f3aa5feea568)

Should be urgently relocated into the **native Settings dialog** here:

![image](https://github.com/user-attachments/assets/58fe63e0-05f2-4a62-93f8-1a83fdb23f50)

If every single plugin developer starts spamming the main toolbar with their own elements, soon the ComfyUI interface will become worse than auto1111, or a Wordpress admin panel with too many plugins installed.

## Important Notice

**Plugin developers do NOT need to spam the GUI with their own buttons; if the plugin is good and its nodes are useful, countless users will install it for sure.**  I honestly believe the only button that has earned the right to be in the main toolbar is the **ComfyUI Manager** extension.

I'll gladly create a PR for this, provided it will be merged. Thank you for the nice plugin BTW.

**Comment by rgthree** [comment]
You can turn off the top menu in the rgthree-comfy settings screen. Just uncheck "**Enable Top Bar Menu**" option.
